lms/djangoapps/teams/csv.py

```diff
@@ -20,12 +21,76 @@ def load_team_membership_csv(course, response):
         response (HttpResponse): Django response object to which
             the CSV content will be written.
     """
-    # This function needs to be implemented (TODO MST-31).
-    _ = course
-    not_implemented_message = (
-        "Team membership CSV download is not yet implemented."
-    )
-    response.write(not_implemented_message + "\n")
+    team_membership_data = _lookup_team_membership_data(course)
+    headers = get_team_membership_csv_headers(course)
+    response.write(','.join(headers) + '\n')
+    for user_data in team_membership_data:
+        row = [user_data.get(header, '') for header in headers]
+        response.write(','.join(row) + '\n')
```

Comment:
Nit, use a `csv.writer`: https://docs.djangoproject.com/en/1.11/howto/outputting-csv/

Reply:
Also, I think `user_data` i going to hold a `dict`, right?  So you can use a `DictWriter`.